### PR TITLE
ADD product_pricelist_item_list_view

### PR DESCRIPTION
**Pull request by eLBati**
This module adds a new menu item "Pricelist rules" to list and search every pricelist rule.
This is useful when you have many rules and you want to search among them.
For instance, if you have many fixed price rules associated to product variants, without this module you could not easly find a variant's price.

### CODE REVIEW

File: product_pricelist_item_list_view/views/pricelist_view.xml [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+                    <group>
+                        <group>
+                            <label for="compute_price" string="Compute Price"/>
+                            <div>
+                                <field name="compute_price" widget="radio"/>
+                                <field name="currency_id" invisible="1"/>
+                                <div attrs="{'invisible':[('compute_price', '!=', 'fixed')]}">
+                                    <field name="fixed_price" nolabel= "1" widget='monetary' options="{'currency_field': 'currency_id'}"/></div>
+                                <div attrs="{'invisible':[('compute_price', '!=', 'percentage')]}">
+                                    <field name="percent_price" nolabel="1" class="oe_inline"/>%%
```
Comment (elicoidal) [inline product_pricelist_item_list_view/views/pricelist_view.xml]: remove %%?

**Comment by elicoidal** [comment]
@eLBati thanks for this very useful module!

File: product_pricelist_item_list_view/views/pricelist_view.xml (hunk 1 above)
Comment (eLBati) [inline product_pricelist_item_list_view/views/pricelist_view.xml]: This is equal to standard `product.pricelist.item` form view: it shows the '%' symbol beside the field

File: product_pricelist_item_list_view/README.rst [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+   :target: http://www.gnu.org/licenses/lgpl-3.0-standalone.html
+   :alt: License: LGPL-3
+
+=========================
+Pricelist rules list view
+=========================
+
+This module adds a new menu item "Pricelist rules" to list and search every pricelist rule.
+This is useful when you have many rules and you want to search among them.
+For instance, if you have many fixed price rules associated to product variants, without this module you could not easly find a variant's price.
```
Comment (tafaRU) [inline product_pricelist_item_list_view/README.rst]: s/easly/easily

**Comment by tafaRU** [comment]
Only one nitpicking in the README.rst otherwise :+1:

**Comment by archetipo** [comment]
👍

File: product_pricelist_item_list_view/views/pricelist_view.xml [hunk 3]
```diff
@@ -0,0 +1,135 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
```
Comment (lasley) [inline product_pricelist_item_list_view/views/pricelist_view.xml]: This can be `<odoo>` w/ no `<data>` el required in v9

**Comment by lasley** [comment]
I am so happy you made this 👍

**Comment by eLBati** [comment]
@tafaRU @lasley thanks, done

**Review by tafaRU — APPROVED** [review]

**Comment by jholze** [comment]
Hey, thanks for this module. Can is ask something ? Do you know a module to link a pricelist item to a customer ?

**Review by lasley — APPROVED** [review]

**Comment by lasley** [comment]
@jholze - I don't know of a module functioning how you mention, and I think it would be rather hard to create. Why not just use the product.pricelist to determine the customers?

**Pull request merged**